main.go

```diff
@@ -42,13 +52,24 @@ func main() {
 			return errors.New("No Vault host provided")
 		}
 
-		if len(cli.String("vault_username")) == 0 {
-			return errors.New("No Vault username provided")
+		if len(cli.String("vault_auth")) == 0 {
```

Comment:
@omerxx why not have a default instead of forcing users to add?

Reply:
You're right. Will do that